app/views/partners/show.html.erb

```diff
@@ -90,6 +90,36 @@
           </div>
         </section>
 
+        <section class="card card-info card-outline" id="partner-users">
+          <div class="card-header">
+            <h2 class="card-title">Users who can access this Partner</h2>
+          </div>
+          <div class="card-body p-0">
+            <div class="tab-content" id="custom-tabs-three-tabContent">
+              <table class="table">
+                <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Email</th>
+                  <th>Invitation Sent</th>
+                  <th>Last Logged In</th>
+                </tr>
+                </thead>
+                <tbody>
+                <% @partner_users.each do |user| %>
+                  <tr>
+                    <td><%= user.name %></td>
+                    <td><%= user.email %></td>
```

Comment:
Does this give you a bullet warning for an N+1 (you may need to test with a few "partner_user" entries)? I think you might need to add an `.includes(profile: :users)` to the `@partner` load in the controller

Reply:
For this line of ruby code in the `partners_controller`:
https://github.com/rubyforgood/human-essentials/pull/2505/commits/343a7896d3824e45e1bac37bd54dad8bcf9a884c#diff-94283ee9fa75fb5564c7731d2d787c3a4046255ffeca480bfd9ab3b47d6b4167R49

I see the following SQL query in the logs when I load the page:
```sql
SELECT "users".* FROM "users" WHERE "users"."partner_id" = $1; --[["partner_id", 2]]
```

Seems like this is not an N+1 issue? Should I add a test for this possible N+1?